crowbar_framework/app/controllers/support_controller.rb

```diff
@@ -103,22 +102,29 @@ def export_supportconfig
 
   def export_chef
     begin
-      Rails.root.join("db").children.each do |file|
-        file.unlink if file.extname == ".json"
-      end
-
-      NodeObject.all.each { |n| n.export }
-      RoleObject.all.each { |r| r.export }
-      Proposal.all.each { |p| p.export }
-
       filename = "crowbar-chef-#{Time.now.strftime("%Y%m%d-%H%M%S")}.tgz"
 
-      pid = Process.fork do
-        exports = Dir.glob(Rails.root.join("db", "*.json").to_s)
-        cmd     = ["tar", "-czf", Rails.root.join("tmp", filename).to_s, *exports]
+      tmpdir = Dir.mktmpdir
+      tmpdirpath = Pathname.new(tmpdir)
```

Comment:
MINOR: i would call it `tmpdir_path` for consistency

or as an alternative just use `tmpdir = Dir.mktmpdir(Pathname.new(tmpdir))` and reference it with `.to_s` where needed